libr/include/r_util/r_strbuf.h

```diff
@@ -6,10 +6,10 @@ extern "C" {
 #endif
 
 typedef struct {
-	char buf[32];
 	int len;
 	char *ptr;
 	int ptrlen;
+	char buf[64];
```

Comment:
why. the reason for this change is because this way strbuf can be casted to char *

Reply:
Yes with the comment above this change is irrelevant now.

Reply:
so shall this be removed?